expo/gbe/views.py

```diff
@@ -1380,8 +1390,12 @@ def view_volunteer(request, volunteer_id):
     volunteer = get_object_or_404(Volunteer, id=volunteer_id)
     if volunteer.profile != request.user.profile:
         validate_perms(request, ('Volunteer Reviewers',), require=True)
-    volunteerform = VolunteerBidForm(instance=volunteer,
-                                     prefix='Volunteer Info')
+    volunteerform = VolunteerBidForm(
+        instance=volunteer,
+        prefix='Volunteer Info', 
+        available_windows=volunteer.conference.windows(),
+        unavailable_windows=volunteer.conference.windows()
```

Comment:
Has a problem - this is optional and volunteers can leave it blank.  When they do, the data doesn't show up in the view case - it looks empty instead of unselected.  Example, hijack becky.suzanne@gmail.com  and view:

http://localhost:8282/volunteer/view/462